README.md

```diff
@@ -22,29 +24,42 @@ must be installed in the runtime environment.
 
 ## Building ##
 
-The library consists of a generic java library (in `lib/`) and a separate Android test project (in `android-test/`).
-The base library jar is built with:
+The library consists of a generic Java library (in `lib/`), a JRE-specific library
+(in `java/`) that includes the generic library, and a separate Android-specific library (in `android/`).
+The JRE-specific library jar is built with:
 
-    gradle lib:jar
+    gradle java:jar
 
 There is also a task to build a fat jar containing the dependencies:
 
-    gradle fullJar
+    gradle java:fullJar
+
+The Android-specific library jar is built with:
+
+    gradle android:assemble
+
+(The `ANDROID_HOME` environment variable must be set appropriately.)
```

Comment:
We have a requirement that if `ANROID_HOME` isn't set, and the Android SDK isn't installed, then it will still build and run the java subproject. Can you check that that still works pls?